### ISSUE TITLE
Investigate why higher amplitude improves keyword_spotting

### ISSUE THREAD

**Issue by tbekolay** [nengo/nengo-loihi#50]

**Comment by drasmuss** [comment]
I know there's been a lot of work on the keyword spotting demo since this came up, do you know if this is still an issue @hunse?

**Comment by hunse** [comment]
I think this is resolved in some of the networks we trained for the keyword spotting paper. However, this has not been incorporated in this repository, as evidenced by this comment: "network was trained with amplitude of 0.002, scaling up improves performance on Loihi" [here](https://github.com/nengo/nengo-loihi/blob/master/docs/examples/keyword_spotting.ipynb).

I think we should take the code from the keyword spotting example and use it for the example here. It includes the training in NengoDL, too, which the current example does not.

**Comment by pblouw** [comment]
I'll add a vote in favor of using the version of the keyword spotter used in the paper as the demo example - it's up-to-date and performs much better. Happy to help with doing the updating if others are busy.

**Comment by drasmuss** [comment]
Yeah that sounds like a good plan, I'll add it to the backlog for a future sprint.
